README.md

```diff
@@ -138,12 +137,99 @@ make test-ts-binary-rococo
 
 Remember to run the clean-up afterwards.
 
-## extend the leasing period
+## License
 
-The default leasing duration for parachain is 1 day, in case you want to extend it (even after it's downgraded to parathread), simply do a `forceLease` via sudo, it should be upgraded to parachain soon again and start to produce blocks.
+GPLv3
 
-![image](https://user-images.githubusercontent.com/7630809/135689832-1f57cd5c-7f83-4fce-9bb0-832b77a38dcc.png)
 
-## License
+## How to Build and Run Parachain and Tee-worker
 
-GPLv3
+## Preparation
+
+- Env: [Setup **SGX TEE** Environment](https://web3builders.notion.site/Setup-SGX-TEE-Environment-68066770831b45b7b632e682cf159477?pvs=4) 
+
+## Build
+
+The `Litentry-worker` needs to be built before the launch of the entire system
+
+```
+git clone https://github.com/litentry/litentry-parachain
+cd ./litentry-parachain/tee-worker
+source /opt/intel/sgxsdk/environment
+SGX_MODE=SW make
+```
+
+## Launch
+
+### 1. Start a local docker setup
+
+In order to create a local docker setup, you can run the following command
+```
+./local-setup/launch.py local-setup/github-action-config-one-worker.json
+```
+This will create three docker containers, 2 Relay Chain Validators, and 1 Parachain Collator. However, it will be using the default ports as present in .env.dev. If you want to run the system by offsetting the default ports by a certain value. You can run this command instead:
+
+```
+./local-setup/launch.py local-setup/development-worker.json local-docker 100
+```
+This will run the same containers and use the offset value of 100.
+
+### 2. Start a local binary setup
+
+In order to create a local binary setup, using default ports, you can run the following command:
+```
+./local-setup/launch.py local-setup/github-action-config-one-worker.json local-binary
+```
+
+If you want to launch the same system by offsetting the port values, you can use this command 
+```
+./local-setup/launch.py local-setup/development-worker.json local-binary 100
+```
+In case you receive the following error:
+```ModuleNotFoundError: No module named 'pycurl'```
+
+Fix it manually by installing pycurl using pip3. 
+
+
+### 3. Remote <> Integritee Node
+
+If you are running the integritee node manually, Then be mindful of the port that is being set while launching the integritee node, For example:
```

Comment:
Shall we use small letters after the comma?

Reply:
Sure, Faisal shared the needed corrections [here](https://www.notion.so/web3builders/How-to-run-parachain-tee-worker-438ffcd144f9494c8b2e0370c2e229a8?d=4b4f5c420a0a43f9addb3748edd73d50#77191e01426749398d911616434c53fe). I'll update this with his replies as soon as I resume and submit the final PR.